lib/rubocop/cop/rspec/rails/minitest_assertions.rb

```diff
@@ -22,30 +24,23 @@ module Rails
         #   expect(b).not_to eq(a)
         #   expect(a).to eq(nil)
         #   expect(a).not_to be_empty
+        #   expect(a).to be_truthy
+        #   expect(b).to be_falsey
         #
         class MinitestAssertions < Base
           extend AutoCorrector
 
           MSG = 'Use `%<prefer>s`.'
-          RESTRICT_ON_SEND = %i[
-            assert_equal
-            assert_not_equal
-            assert_instance_of
-            assert_not_instance_of
-            assert_includes
-            assert_not_includes
-            assert_match
-            assert_nil
-            assert_not_nil
-            assert_empty
-            assert_not_empty
-            refute_equal
-            refute_instance_of
-            refute_includes
-            refute_nil
-            refute_empty
-            refute_match
+          ASSERT_SUFFIXES = %w[
+            equal not_equal instance_of not_instance_of includes not_includes
+            match nil not_nil empty not_empty true false
           ].freeze
+          REFUTE_SUFFIXES = %w[
+            equal instance_of includes nil empty match
```

Comment:
This decreases discoverability.
Are we hitting the line limit for this cop? What about grouping assertions per sub-cop (eg one line for “equal” etc)?

Reply:
I've opened #1790 which goes in another approach to this that should retain the readability